runner.py

```diff
@@ -54,6 +55,12 @@ def run_agent(terminate_event, callback_event, config_file, name, team, index, m
 
 
 def main():
+    # Inject DLL
+    injector_dir = os.path.join(os.path.dirname(__file__), "RLBot_Injector.exe")
+    incode=subprocess.call([injector_dir, 'hidden'])
+    injector_codes=['INJECTION_SUCCESSFUL','INJECTION_FAILED','MULTIPLE_ROCKET_LEAGUE_PROCESSES_FOUND','RLBOT_DLL_ALREADY_INJECTED','RLBOT_DLL_NOT_FOUND','MULTIPLE_RLBOT_DLL_FILES_FOUND']
```

Comment:
We should fail on some of these codes.

Reply:
what do you mean?

Reply:
INJECTION_SUCCESSFUL or RLBOT_DLL_ALREADY_INJECTED seem like success statuses. The others should probably terminate runner. Also this code will likely change with the GUI in v4 branch so might be better to implement as part of v4.

Reply:
Fixed (i believe)